src/components/NewProfile.js

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from "react";
+import { Component } from "react";
+import './styles.css'
+
+// Color Palette
```

Comment:
I love how you have all the theme's colours and formatting listed here! really makes it easier for future formatting and code adjustments as well

Reply:
that was actually @amywangg gotta give her credit LOL but thank you I will definitely keep in mind to keep colours for future codes